app/components/works/affiliations_row_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Works
+  # Add a single affiliation to an author / contributor.
+  class AffiliationsRowComponent < ApplicationComponent
+    def initialize(form:)
+      @form = form
+    end
+
+    attr_reader :form
+
+    def error?
+      errors.present?
```

Comment:
I don't see this called anywhere explicitly. Wired up automagically elsewhere?

Reply:
Copypasta. Removed.